raiden/transfer/log.py

```diff
@@ -133,10 +149,17 @@ def get_all_state_changes(self):
         )
         return result.fetchall()
 
+    def get_all_state_events(self):
+        cursor = self.conn.cursor()
+        result = cursor.execute(
+            'SELECT * from state_events'
+        )
+        return result.fetchall()
+
     def read(self):
         pass
 
-    def last_identifier(self):
+    def last_state_change_id(self):
```

Comment:
This is only be safe inside a transaction with the table's write lock acquired, otherwise it's not thread safe, IMHO this should be removed.